src/codesyncd/utils.ts

```diff
@@ -205,7 +205,7 @@ export class statusBarMsgs {
 		} catch (e) {
 			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 			// @ts-ignore
-			putLogEvent(e.stack);
+			CodeSyncLogger.error(e.stack);
```

Comment:
Can we add a description of the error message here? to make this error more readable in the logs.